scripts/compare-prs.py

```
@@ -0,0 +1,175 @@
+            continue
+
+        # Both exist - check if binary (content is None)
+        if content1 is None or content2 is None:
+            binary_files.append(filepath)
+            continue
+
+        if content1 == content2:
+            identical.append(filepath)
+        else:
+            # Check if differences are only static_cast style
+            lines1 = [normalize_line(l) for l in content1.split('\n')]
```

Ruff (0.15.6)

[error] Ambiguous variable name: `l`

(E741)